libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -163,6 +162,21 @@ class MapboxNavigation(
     private var rerouteController: RerouteController?
     private val defaultRerouteController: RerouteController
 
+    /**
+     * Starts listening for location updates and enters an `Active Guidance` state if there's a primary route available
+     * or a `Free Drive` state otherwise.
+     *
+     * @see [registerTripSessionStateObserver]
+     * @see [registerRouteProgressObserver]
+     */
+    @RequiresPermission(anyOf = [ACCESS_COARSE_LOCATION, ACCESS_FINE_LOCATION])
+    fun startTripSession() {
+        tripSession.start()
+        notificationChannelField?.let {
+            monitorNotificationActionButton(it.get(null) as ReceiveChannel<NotificationAction>)
+        }
+    }
```

Comment:
Could this be reverted to where it was before https://github.com/mapbox/mapbox-navigation-android/pull/3800? Why was it moved before `init`? 🤔

Reply:
🤔  .. uhh, rebase?

Reply:
<command><shift><up>

🤦 . must have pulled it up when copying things around